package.json

```diff
@@ -18,23 +19,21 @@
   },
   "homepage": "https://github.com/BrianMitchL/pr-chatter-action#readme",
   "dependencies": {
-    "@actions/core": "^1.2.6",
-    "@actions/github": "^4.0.0",
-    "node-fetch": "^2.6.1",
+    "@actions/core": "^1.6.0",
+    "@actions/github": "^5.0.0",
+    "node-fetch": "^2.6.6",
     "sentiment": "^5.0.2"
   },
   "devDependencies": {
-    "@types/node-fetch": "^2.5.8",
-    "@vercel/ncc": "^0.27.0",
-    "husky": "^4.3.8",
```

Comment:
No more husky?

Reply:
Byeee! `simple-git-hooks` does the job great and is way less overhead to manage.